process/block/metablock.go

```diff
@@ -1637,12 +1641,6 @@ func (mp *metaProcessor) ApplyBodyToHeader(hdr data.HeaderHandler, bodyHandler d
 
 	metaHdr.ValidatorStatsRootHash = rootHash
 
-	epochStart, err := mp.createEpochStartForMetablock()
```

Comment:
This was moved to CreateNewHeader, because it is needed earlier?

Reply:
yes, we would need this early for example when we change the validation for epoch genesis block to be done by the new epoch validators.

Reply:
ok